src/cohortfinder_choosehappy/cohortfinder_colormod_original.py

```diff
@@ -327,10 +398,15 @@ def runCohortFinder(args):
             for fname in fnamessub:
                 print(fname)
                 fullfname = glob.glob(f"{basedir}/**/{fname}*thumb*small*")
+                if (len(fullfname) ==0):
+                    error_message = f"There is no thumbnail images in the subfolders of {basedir}. Please check!"
+                    logging.error(error_message)
+                    raise ValueError(error_message)
                 # print(hqc_results_tsv)
-                print(f"This is the filename name: {basedir}")
-                io = cv2.cvtColor(cv2.imread(fullfname[0]), cv2.COLOR_BGR2RGB)
-                axs.pop().imshow(io)
+                else:
+                    print(f"This is the filename name: {basedir}")

```

Comment:
ctrl + f for "print" and replace with logging.info(). 

Also, log fullfname[0] instead of basedir